hw/system/occamy/src/occamy_soc_ctrl/occamy_soc_reg.hjson.tpl

```diff
@@ -0,0 +1,250 @@
+    }
+    { multireg:
+      { name: "PAD"
+        desc: "GPIO pad configuration."
+        swaccess: "rw"
+        hwaccess: "hro",
+        count: "NumPads"
+        cname: "pad"
+        fields: [
+          { bits: "0"
+            name: "SLW"
+            resval: "0"
+            desc: '''
+                    Slew control.
+                    1: when VDDIO = 1.5/1.2V
+                    0: when VDDIO = 1.8V
+                  '''
+          },
+          { bits: "1"
+            name: "SMT"
+            resval: "0"
+            desc: "Active high Schmitt Trigger enable."
+          },
+          { bits: "3:2"
+            name: "DRV"
+            resval: "2"
+            desc: "Drive strength."
+          }
+        ]
+      }
+    }
+    { multireg:
+      { name: "ISOLATE"
+        desc: "Isolate port of given quadrant."
+        swaccess: "rw"
+        hwaccess: "hro"
+        count: "NumS1Quadrants"
+        cname: "isolate"
+        fields: [
+          { bits: "3:0"
+            resval: "1"
+            name: "ISOLATE"
+            desc: '''
+                  Isolate S1 Quadrant. Four bits corresponding to:
+                    - Bit 0: Narrow In
+                    - Bit 1: Narrow Out
+                    - Bit 2: Wide In
+                    - Bit 3: Wide Out
+
+                  0: De-isolate request
+                  1: Isolate request
+                  '''
+          }
+        ]
+      }
+    },
+    { multireg:
+      { name: "ISOLATED"
+        desc: "Isolation status of S1 quadrant and port"
+        swaccess: "ro"
+        hwaccess: "hwo"
+        hwqe:     "true",
+        hwext:    "true",
+        count: "NumS1Quadrants"
+        cname: "isolated"
+        fields: [
+          { bits: "3:0"
+            resval: "1"
+            name: "ISOLATED"
+            desc: '''
+                  Isolate satus of S1 Quadrant. Four bits corresponding to:
+                    - Bit 0: Narrow In
+                    - Bit 1: Narrow Out
+                    - Bit 2: Wide In
+                    - Bit 3: Wide Out
+
+                  Isolation status:
+                    0: De-isolated
+                    1: Isolated
+                  '''
+          }
+        ]
+      }
+    },
+    { multireg:
+      { name: "RO_CACHE_ENABLE",
+        desc: "Enable read-only cache of quadrant.",
+        swaccess: "rw"
+        hwaccess: "hro"
+        count: "NumS1Quadrants"
+        cname: "ro_enable"
+        fields: [
+          { bits: "0:0"
+            resval: "0"
+            name: "ENABLE"
+            desc: "Enable RO cache of S1 quadrant."
+          }
+        ]
+      },
+    },
+    { multireg:
+      { name: "RO_CACHE_FLUSH",
+        desc: "Flush read-only cache.",
+        swaccess: "rw"
+        hwaccess: "hrw"
+        count: "NumS1Quadrants"
+        cname: "ro_flush"
+        fields: [
+          { bits: "0:0"
+            resval: "0"
+            name: "FLUSH"
+            desc: "Flush (inv RO cache of S1 quadrant."
```

Comment:
I guess the missing closing parenthesis comes from here.

Reply:
Yes, thanks!